### DISCUSSION TITLE
Cannot use mdast-util-from-markdown as es module export in electron, possible to get it with cjs?

### DISCUSSION THREAD

**Discussion by nikivdev** [syntax-tree/unist#108] [Q&A] [answered]
<img width="504" alt="image" src="https://github.com/syntax-tree/unist/assets/6391776/d633ac56-01b4-4239-95c6-6b22483d5737">

**Comment by ChristianMurphy** [comment, marked as answer]
Welcome @nikitavoloboev! 👋

> Cannot use mdast-util-from-markdown as es module export in electron

See https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c#how-can-i-import-esm-in-electron
You have options:
1. Help out with https://github.com/electron/electron/pull/37535 so Electron does support ESM
2. Bundle your dependencies with Webpack/Rollup/ESBuild into a CommonJS bundle
3. Use the [esm](https://github.com/standard-things/esm) package.

> possible to get it with cjs?

`unified`, `mdast-util-from-markdown`, and the ecosystem of packages target ES2020 and ESM (both supported by Node 16 LTS).
You are welcome to transpile/down level that into earlier versions of JavaScript, or any bespoke/legacy module format you like using build tools like `webpack`, `rollup`, `babel`, `vite`, `swc`, etc.

**Reply by nikivdev** [reply]
Thank you
